pkg/api/v1/events/handlers.go

```diff
@@ -86,8 +86,9 @@ func (h *Handler) UpdateCondition(ctx context.Context, updEvt *v1types.Condition
 	if err != nil {
 		if errors.Is(err, store.ErrConditionNotFound) {
 			h.logger.WithFields(logrus.Fields{
-				"serverID":      updEvt.ConditionUpdate.ServerID,
-				"conditionKind": updEvt.Kind,
+				"server_id":      updEvt.ConditionUpdate.ServerID,
```

Comment:
can this be done throughout the code base so its consistent - `s/\"serverID\"/\"server_id\"/g` and `s/\"conditionID\"/\"condition_id\"/g`?

Reply:
We can but not in this pull-request which I want to keep targeted to the issue at hand.